api/camunda/generation-strategy.js

```diff
@@ -4,7 +4,18 @@ const outputDir = "docs/apis-tools/camunda-api-rest/specifications";
 const specFile = "api/camunda/camunda-openapi.yaml";
 
 function preGenerateDocs() {
-  hackChangesetDescription();
```

Comment:
This description hack is no longer needed!

Reply:
🥳